libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -22,6 +22,7 @@ namespace Microsoft.Bot.Builder.LanguageGeneration
     public class Templates : List<Template>
     {
         private readonly string newLine = "\r\n";
```

Comment:
>"\r\n" [](start = 42, length = 6)

Have we standardized on Windows end of line?  What happens if people create templates on a Mac?

Reply:
Evaluation process of lg is platform unrelated, because LG file can be created from any platform, and evaluate from another platform. So, change `newLine = "\r\n"` to `Environment.NewLine` is also ok.

Reply:
changed to Environment.NewLine to make sure the code behaves consitently on all platforms